src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -43,14 +43,78 @@
+ * <p>The GATK4 Best Practice Workflow for SNP and Indel calling uses GenomicsDBImport to merge GVCFs from multiple samples.
+ * GenomicsDBImport offers the same functionality as CombineGVCFs and comes from the <i>Intel-Broad Center for Genomics</i>.
+ * The datastore transposes sample-centric variant information across genomic loci to make data more accessible to tools.
+ * </p>
+ *
+ * <p>To query the contents of the GenomicsDB datastore, use
+ * <a href='https://software.broadinstitute.org/gatk/documentation/tooldocs/current/org_broadinstitute_gatk_tools_walkers_variantutils_SelectVariants.php'>SelectVariants</a>.
+ * See <a href='https://software.broadinstitute.org/gatk/documentation/article?id=10061'>Tutorial#10061</a> to get started. </p>
+ *
+ * <p>Details on GenomicsDB are at
+ * <a href='https://github.com/Intel-HLS/GenomicsDB/wiki'>https://github.com/Intel-HLS/GenomicsDB/wiki</a>.
+ * In brief, GenomicsDB is a utility built on top of TileDB. TileDB is a format for efficiently representing sparse data.
+ * Genomics data is typically sparse in that each sample has few variants with respect to the entire reference genome.
+ * GenomicsDB contains code to specialize TileDB for genomics applications, such as VCF parsing and INFO field annotation
+ * calculation.
+ * </p>
+ *
+ * <h3>Input</h3>
+ * <p>
+ * One or more GVCFs produced by in HaplotypeCaller with the `-ERC GVCF` or `-ERC BP_RESOLUTION` settings, containing
+ * the samples to joint-genotype.
+ * </p>
+ *
+ * <h3>Output</h3>
+ * <p>
+ * A GenomicsDB workspace
+ * </p>
+ *
+ *  <h3>Usage examples</h3>
+ *
+ *  Provide each sample GVCF separately.
+ *  <pre>
+ *    gatk GenomicsDBImport \
+ *      -V data/gvcfs/mother.g.vcf.gz \
+ *      -V data/gvcfs/father.g.vcf.gz \
+ *      -V data/gvcfs/son.g.vcf.gz \
+ *      -genomicsDBWorkspace my_database \
+ *      --intervals 20
+ *  </pre>
+ *
+ *  Provide sample GVCFs in a map file.
+ *
+ *  <pre>
+ *    gatk --javaOptions "-Xmx4g -Xms4g" \
+ *       GenomicsDBImport \
+ *       -genomicsDBWorkspace my_database \
+ *       -batchSize 50 \
+ *       -L chr1:1000-10000 \
+ *       -sampleNameMap cohort.sample_map \
+ *       --reader-threads 5
+ *  </pre>
+ *
+ *  The sample map is a tab-delimited text file with sample_name--tab--path_to_sample_vcf per line.
+ *
+ *  <pre>
+ *  sample1      sample1.vcf.gz
+ *  sample2      sample2.vcf.gz
+ *  sample3      sample3.vcf.gz
+ *  </pre>
+ *
+ * <h3>Caveats</h3>
+ * <ul>
+ *     <li>A single interval must be provided. This means each import is limited to a maximum of one contig</li>
+ *     <li>Currently, only supports diploid data</li>
+ *     <li>Input GVCFs cannot contain multiple entries for a single genomic position</li>
+ *     <li>Currently, a GenomicsDB must be used from the directory structure in which it was created. If copied between file systems, the absolute path must be the same at the destination.</li>
```

Comment:
A few more caveats: 

* The --genomicsdb-workspace-path must point to a non-existent or empty directory.
* The -Xmx value the tool is run with should be less than the total amount of physical memory available by at least a few GB, as the native TileDB library requires additional memory on top of the Java memory (this one is really important, as failure to leave enough memory for the native code can result in confusing error messages!)